api/src/main/resources/liquibase-update-to-latest.xml

```diff
@@ -796,6 +796,130 @@
 			<column name="uuid" type="char(38)" />
 		</addColumn>
 	</changeSet>
+	<changeSet author="vshankar" id="201609171146-2">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<columnExists tableName="cohort_member" columnName="cohort_member_id"/>
+			</not>
+		</preConditions>
+		<addColumn tableName="cohort_member">
+			<column name="cohort_member_id" type="int">
+				<constraints nullable="false"/>
+			</column>
+		</addColumn>
+	</changeSet>
+	<changeSet author="vshankar" id="201609171146-3">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<columnExists tableName="cohort_member" columnName="start_date"/>
+			</not>
+		</preConditions>
+		<addColumn tableName="cohort_member">
+			<column name="start_date" type="datetime">
+				<constraints nullable="false"/>
+			</column>
+		</addColumn>
+	</changeSet>
+	<changeSet author="vshankar" id="201609171146-4">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<columnExists tableName="cohort_member" columnName="end_date"/>
+			</not>
+		</preConditions>
+		<addColumn tableName="cohort_member">
+			<column name="end_date" type="datetime"/>
+		</addColumn>
+	</changeSet>
+	<changeSet author="vshankar" id="201609171146-5">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<columnExists tableName="cohort_member" columnName="creator"/>
+			</not>
+		</preConditions>
+		<addColumn tableName="cohort_member">
+			<column name="creator" type="int">
+				<constraints nullable="false"/>
+			</column>
+		</addColumn>
+		<addForeignKeyConstraint baseColumnNames="creator" baseTableName="cohort_member" constraintName="cohort_member_creator" deferrable="false" initiallyDeferred="false" referencedColumnNames="user_id" referencedTableName="users"/>
+	</changeSet>
+	<changeSet author="vshankar" id="201609171146-6">
+		<preConditions>
+			<not>
+				<primaryKeyExists tableName="cohort_member"/>
+			</not>
+		</preConditions>
+		<addPrimaryKey tableName="cohort_member" columnNames="cohort_member_id"/>
+	</changeSet>
+	<changeSet author="vshankar" id="201609171246-7">
+		<addAutoIncrement tableName="cohort_member" columnName="cohort_member_id" columnDataType="int"/>
+	</changeSet>
+	<changeSet id="201610042145-1" author="vshankar">
+        <preConditions>
+			<not>
+				<columnExists tableName="cohort_member" columnName="date_created"/>
+			</not>
+		</preConditions>
+        <addColumn tableName="cohort_member">
+			<column name="date_created" type="DATETIME">
+				<constraints nullable="false"/>
```

Comment:
Same here, you will need to insert values for existing columns and then add the constraint, I think dateCreated can be that of the cohort for existing rows